docs/specs/circuit.md

```diff
@@ -3,210 +3,67 @@
 We build our virtual machines in a STARK proving system with a multi-matrix commitment scheme and shared verifier
 randomness between AIR matrices to enable permutation arguments such as log-up.
 
-In the following, we will refer to a circuit as a collection of AIR matrices (also referred to as chips) of possibly
-different heights, which may communicate with one another over buses using a log-up permutation argument. We refer to
-messages sent to such a bus as [interactions](https://github.com/openvm-org/stark-backend/tree/main/crates/stark-backend/src/interaction).
+In the following, we will refer to a circuit as a collection of chips that communicate with one another over buses using a LogUp permutation argument. We refer to messages sent to such a bus as [interactions](https://github.com/openvm-org/stark-backend/blob/main/docs/interactions.md). Every _chip_ is an entity responsible for a certain operation (or set of operations), and it has an AIR to check the correctness of its execution.
```

Comment:
technically, in the backend we used "interaction" to refer to the "for every row, send ..." constraint/rule. But we do also colloquially use interaction to kind of refer to the message. I couldn't find a good way to really distinguish this, and here when you say refer to mesage**s** -> interaction, it is technically still correct in a sense. so no change required